doc/authors_emeritus.rst

```diff
@@ -0,0 +1,33 @@
+- Mathieu Blondel
```

Comment:
Could we keep these ordered alphabetically by last name?

Reply:
You mean by first name ?